script/cmd/ColdStorageAddressBookPlugin

```diff
@@ -0,0 +1 @@
9480945061fffc92506002915016931691161717905555600190565b6001811615908161176f575b50156117645738611705565b509496505050505050565b9050151538611758565b5093955050505050565b5083156116f4565b6001600160a01b0391821681529116602082015260408101919091526080606082018190526112cb92910190610e59565b9181156118255780516118065750506001600160a01b038116156117dd5790565b6040516375f4229560e11b81523360048201526001600160a01b03919091166024820152604490fd5b604051630c4f82d960e41b8152918291610d079185336004860161178b565b91803b1561188257505061183881611999565b906001600160a01b039081831615611871575b8183161561185f575b508116156117dd5790565b61186a919250611aaf565b9038611854565b915061187c82611a3a565b9161184b565b610d07836040519384936347662f4160e01b8552336004860161178b565b60408051338152637c9c64bb60e11b60208201526000918101919091526080916bffffffffffffffffffffffff199060601b1660608201522054151590565b80519060005b8281106118f157505050565b6119176bffffffffffffffffffffffff1961190c8385611276565b5160601b1633611b53565b15611924576001016118e5565b610d07916001600160a01b039161193a91611276565b51604051635b01bbf160e01b815233600482015291166001600160a01b031660248201529081906044820190565b906020825192015163ffffffff60e01b90818116936004811061198a57505050565b60040360031b82901b16169150565b6001600160e01b03196119ab82611968565b1663a9059cbb60e01b81148015611a2a575b8015611a1a575b8015611a0a575b156119e7575060448151106119e1576024015190565b50600090565b6323b872dd60e01b146119fa5750600090565b60648151106119e1576044015190565b5063a457c2d760e01b81146119cb565b50633950935160e01b81146119c4565b5063095ea7b360e01b81146119bd565b6001600160e01b0319611a4c82611968565b1663a22cb46560e01b8103611a6c575060448151106119e1576024015190565b637921219560e11b8103611a8b575060c48151106119e1576044015190565b631759616b60e11b14611a9e5750600090565b6101048151106119e1576044015190565b6001600160e01b0319611ac182611968565b1663a22cb46560e01b81148015611b43575b15611ae9575060448151106119e1576024015190565b632142170760e11b81148015611b33575b15611b10575060648151106119e1576044015190565b635c46a7ef60e11b14611b235750600090565b60a48151106119e1576044015190565b506323b872dd60e01b8114611afa565b5063095ea7b360e01b8114611ad3565b9061ffff1916908115611bdb57604080516001600160a01b039092168252637c9c64bb60e11b602083015260009082015260608101828152608082208054909290611bd257600160809252209182549283158015611bc8575b15611bbc57600193505555600190565b90600217905555600190565b5060018416611bac565b50505050600090565b5050600090565b906040519160018060a01b03168252637c9c64bb60e11b602083015260008060408401526000906060840160019460018252608090608081205492604051948860808701995b611c45575b5050505050506001820160051b016080016040528252565b9091929394968188161580611c9b575b15611c955761fffe198816908201600581901b880187018290529760021615611c8c5783528085852054915b959493929195611c28565b50808291611c81565b96611c2d565b50871515611c55565b805160005b818110611cb557505090565b80611cc260019285611276565b5160601c611cd08286611276565b5201611ca956fea26469706673582212209a2412a8aeab08683f5199b932bfa3c1fa159ee5c0a0a0e0d878a0bd2f99505664736f6c63430008180033
```

Comment:
I see all the runs on all chains have json output - does it mean you did forge deploy for all contracts ? Were we able to test these cast commands on any chain?

Reply:
Yes, I used forge deploy for all of the contracts across all of the chains, so I have not tested these commands. But, I did construct them using the instructions specified in the README. Should I test them on a local network to confirm I get the same address?

Reply:
Yeah I usually forge deploy with 1 chain and use this for rest of the chains - as testing for chain expansion later

Reply:
Just curious, does cast send also verify the contracts like forge deploy?

Reply:
It has this option to https://book.getfoundry.sh/reference/cast/cast-send#etherscan-options